### PR TITLE
ARM64: Enable a gcstress private job

### PR DESCRIPTION
**Pull request by kyulee1**

### CODE REVIEW

**Comment by kyulee1** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -506,7 +506,15 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                 // GCStress=C is currently not supported on OS X
                 if (os != 'CentOS7.1' && os != 'OSX' && !(os in bidailyCrossList)) {
                     assert (os == 'Windows_NT') || (os in Constants.crossList)
-                    Utilities.addPeriodicTrigger(job, '@weekly')
+                    if (architecture == 'arm64') {
+                        assert (os == 'Windows_NT')
```
Comment (jashook) [inline netci.groovy]: Nit: assert is unnecessary it is already tested on 508.

**Comment by jashook** [comment]
lgtm

File: netci.groovy (hunk 1 above)
Comment (kyulee1) [inline netci.groovy]: Well. This is intentional. arm64 is enabled only for Windows not cross OSes, yet.

**Pull request merged**